scipy/linalg/decomp_svd.py

```diff
@@ -279,6 +280,97 @@ def diagsvd(s, M, N):
         raise ValueError("Length of s must be M or N.")
 
 
+def hosvd(a, full_tensor=True, check_finite=True):
+    """
+    Higher-order SVD (HOSVD)
+
+    Factorizes the M-D tensor `a` into a list of M matrices ``U_k`` containing
+    the left singular vectors of the unfolded tensor with respect to `k`\\ th
+    axis and a core tensor ``S`` such that
+    :math:`a = (U_1, U_2, \\dots, U_M) \\cdot S`.
+
+    Parameters
+    ----------
+    a : array_like
+        Tensor to decompose
+    full_tensor : bool, optional
+        If True (default), `S` is of the same shape as the input tensor `a` and
+        all `U_k` are of the shape ``(n, n)`` with `n` corresponding to the
+        `k`\\ th axis of `S`.
+        If False, the shapes of the `U_k` are ``(n, m)`` with `n` corresponding
+        to the size of the `k`\\ th axis of the input `a` and `m` to the maybe
```

Comment:
Single ticks for parameters from the function itself and for other non code notations like the following, double ticks.
```suggestion
        If True (default), ``S`` is of the same shape as the input tensor `a` and
        all ``U_k`` are of the shape ``(n, n)`` with ``n`` corresponding to the
        `k`\\ th axis of ``S``.
        If False, the shapes of the `U_k` are ``(n, m)`` with ``n`` corresponding
        to the size of the `k`\\ th axis of the input `a` and ``m`` to the maybe
```